src/app/utils/testing.utils.ts

```diff
@@ -336,5 +358,30 @@ function mockBodyType(args?: Partial<BodyType>): BodyType {
   const mock: BodyType = {
     description: 'the first letter should be capital'
   } as BodyType;
+
+  return { ...mock, ...args };
+}
+
+function mockPSVBrakes(args?: Partial<Brakes>): Brakes {
+  const mock: Brakes = {
+    brakeCodeOriginal: '123',
+    brakeCode: '145202',
+    retarderBrakeOne: 'electric',
+    brakeForceWheelsNotLocked: {
```

Comment:
this should have its own mock method

Reply:
it's part of the same accordion and everything. there can't be used outside of that component. shall i still split those ?